server/pulp/server/controllers/repository.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+from pulp.common import dateutils
+from pulp.plugins.loader import api as plugin_api
+from pulp.server.db import connection, model
+
+_logger = logging.getLogger(__name__)
+
+
+def find_repo_content_units(
+        repository, repo_content_unit_q=None,
+        units_q=None, units_fields=None, limit=None, skip=None):
+    """
+    Search content units associated with a given repository
+
+    Each association returned will have a parameter "unit" set to the actual unit
```

Comment:
Should `parameter` be `attribute`? I'm not sure how to set a parameter of a return call without currying the object.